test/services/notifications/setup/batch-notifications.service.test.js

```diff
@@ -37,6 +36,10 @@ describe('Notifications Setup - Batch notifications service', () => {
     // notifications still saved in the database regardless of batch size
     NotifyConfig.batchSize = 1
 
+    // By setting the delay to 100ms we can keep the tests fast whilst assuring our batch mechanism is delaying
+    // correctly, we do not want increase the timeout for the test as we want them to fail if a timeout occurs
+    NotifyConfig.delay = ONE_HUNDRED_MILLISECONDS
```

Comment:
Apologies, this applies to the `batchSize` as well (didn't clock it on first pass).

We normally stub changes to config. See `test/presenters/bill-runs/review/base-review.presenter.test.js` for an example.

```javascript
// Things we need to stub
const NotifyConfig = require('../../../../config/notify.config.js')

describe('Awesome service', () => {
  beforeEach(() => {
    Sinon.stub(NotifyConfig, 'batchSize').value(1)
    Sinon.stub(NotifyConfig, 'delay').value(ONE_HUNDRED_MILLISECONDS)
  })

  afterEach(() => {
    Sinon.restore()
  })
})